_drafts/EH-tidyx.md

```diff
@@ -0,0 +1,25 @@
+
+To keep his skills sharp Ellis regularly takes part in Tidy Tuesday, a project out of the R for Data Science Online Learning Community. The goal of Tidy Tuesday is to provide a constructivce and supportive space for new and seasoned R users alike to practice their data wrangling and visualization skills. The premise is that each week a dataset is shared out via the [TidyTuesday github repository](https://github.com/rfordatascience/tidytuesday). The dataset has generally been "tamed", meaning that some cleaning has been done, but it doesn't meet the standards of being "tidy". [Tidy data](https://cran.r-project.org/web/packages/tidyr/vignettes/tidy-data.html#:~:text=Tidy%20data%20is%20a%20standard,Each%20observation%20forms%20a%20row.) is a term coined by Hadley Wickham. The principles behind tidy data act to provide a standard way of organizing data values in a dataset to facilitate analysis. Participants are asked to use techniques from the (R for Data Science book)[https://r4ds.had.co.nz/] to explore and tidy the dataset and finally create a visualization to share out to the R statistics community via Twitter using the hashtag [#TidyTuesday](https://twitter.com/search?q=%23TidyTuesday&src=typeahead_click).
+
+Having participated in Tidy Tuesday and seen the benefits, Ellis wanted to know why R users were not taking advantage of this free, weekly event! To get some answers he conducted a Twitter poll asking what were the main reasons that kept folks from taking part in Tidy Tuesday. The top two answers were lack of time and newcomers felt nervous and did not know where to start. While he couldn't really change the fact that some folks don't have time to participate, he definitely could find a way to reduce the barrier of entry for Tidy Tuesday. And so TidyX was born!
+
+[TidyX](https://www.youtube.com/channel/UCP8l94xtoemCH_GxByvTuFQ/videos) is a screencast with goal of providing concrete examples of how to interact with a Tidy Tuesday dataset. In each episode Ellis and his co-host, Patrick Ward, work through the code of a Tidy Tuesday submission from a pervious week. Code review is a great way to get ideas on how to explore a dataset and learn about new methods for data wrangling and visualization. Then Ellis shows viewers how the same code can be applied to a different dataset to emphasize how code can be reused between datasets with similar features.
+
+We are big fans of Tidy Tuesday here at The Coop and highly recommend giving it a try! And now, if you need some inspiration there is the TidyX screencast! In the past, Coop community members had taken part in Tidy Tuesday through the Data Visualization community group. While that group is on haitus, we still encourage anyone taking part in Tidy Tuesday to post their work in the #data-viz channel on The Coop Slack!
+
+If your or someone you know is working on a project that you think should be highlighted - let us know by DMing me on Slack or Teams!
```

Comment:
I'd change this to be more accessible: let us know via Slack, MS Teams, or emailing coophelp